api/src/main/resources/rdr.yaml

```diff
@@ -0,0 +1,202 @@
+      profile: profile authorization
+
+security:
+  - googleoauth:
+    - openid
+    - email
+    - profile
+    - https://www.googleapis.com/auth/devstorage.full_control
+
+produces:
+  - application/json
+
+##########################################################################################
+## PATHS
+##########################################################################################
+
+paths:
+  /workbench/directory/workspaces:
+    post:
+      description: Submits one or more workspace snapshots for storage in the RDR.
+      tags:
+        - RDR
+      consumes:
+        - application/json
+      operationId: sendWorkspace
+      parameters:
+        - in: body
+          required: true
+          name: workspaces
+          schema:
+            type: array
+            items:
+              $ref: "#/definitions/RDRWorkspace"
+      responses:
+        200:
+          description: successful operation
+
+  /workbench/directory/researchers:
+    post:
+      description: Submits one or more researcher snapshots for storage in the RDR.
+      tags:
+        - RDR
+      operationId: sendResearcher
+      consumes:
+        - application/json
+      parameters:
+        - in: body
+          required: true
+          name: researchers
+          schema:
+            type: array
+            items:
+              $ref: "#/definitions/RDRResearcher"
+      responses:
+        200:
+          description: successful operation
+
+
+
+definitions:
+  RDRWorkspace:
+    type: object
+    properties:
+      workspaceId:
+        type: integer
+      name:
+        type: string
+      creationTime:
+        type: string
+        format: date-time
+      modifiedTime:
+        type: string
+        format: date-time
+      status:
+        type: string
+        description: Workspace Status
+        enum:
+          - ACTIVE
+          - INACTIVE
+      workspaceUsers:
+        type: array
+        items:
+          $ref: '#/definitions/RDRWorkspaceUser'
+      excludeFromPublicDirectory:
+        type: boolean
+      diseaseFocusedResearch:
+        type: boolean
+      diseaseFocusedResearchName:
+        type: string
+      otherPurposeDetails:
+        type: string
+      methodsDevelopment:
+        type: boolean
+      controlSet:
+        type: boolean
+      ancestry:
+        type: boolean
+      socialBehavioral:
+        type: boolean
+      populationHealth:
+        type: boolean
+      drugDevelopment:
+        type: boolean
+      commercialPurpose:
+        type: boolean
+      educational:
+        type: boolean
+      otherPurpose:
+        type: boolean
+
+  RDRWorkspaceUser:
+    type: object
+    properties:
+      userId:
+        type: integer
+      role:
+        type: string
+        enum:
+          - READER
+          - WRITER
+          - OWNER
+      status:
+        type: string
+        description: Workspace User Status
+        enum:
+          - ACTIVE
+          - INACTIVE
+
+  RDRResearcher:
```

Comment:
Maybe I missed it in the design doc, but we will also need an "excludeFromPublicDirectory" field on the researcher level, to allow admins to prevent non-end-user objects from being published to the directory. That may require some follow-up with the RDR team, so it's worth tracking as a separate small ticket.